### ISSUE TITLE
"Select all" doesn't actually "Select all"

### ISSUE THREAD

**Issue by jikquantopian** [bitwarden/web#303]
At https://vault.bitwarden.com/#/vault, while viewing "All Items", I click on the gear and select "Select All", and it doesn't actually select all. It stops about 3/4 of the way down.

**Comment by jikquantopian** [comment]
I think it will only select up to 500 items at a time? Ugh.

**Comment by kspearrin** [comment]
It should max out at 500. This is by design.

**Comment by jikquantopian** [comment]
So, here's the thing.... I just imported like 700 items into my personal vault. I want to move them all into an organization to which I have access. "No problem," I say to myself, "I'll just select them all and then select Share."

Nope, can't do that, can only select 500.

So, find, I select 500 and move them. It tells me that 499 can be moved and one can't. It doesn't tell me which one can't or why. Fine, so I tell it to go ahead and move the 499.

So now that's done, and 499 items have been moved into the organization. And yet they're still listed in the "All Items" view, and there is, as far as I can tell, no way to view only items that are in my private vault and not in an organization.

So now I have to scroll through the list of 700+ looking for the ones without the little "shared" icon next to them telling me that they're in an organization. And fine, I know that most of them are at the bottom, and I can scroll down to the end and click them all one by one. But I can't "Select All" them, because of the aforementioned limit and the fact that there's no way I can see to limit my view to just items that are in my personal vault.

And that doesn't help with the 1 out of 500 which for some reason could not be shared. To find that, I have to actually scroll down and look at every item one by one to see if I can spot the one out of 500 that doesn't have a little icon next to it.

This is a sub-optimal user experience, to say the least.
